api/jobs.go

```diff
@@ -1,8 +1,6 @@
 package api
 
-import (
-	"github.com/pborman/uuid"
-)
+import "github.com/pborman/uuid"
```

Comment:
Odd.  `go fmt` did not do this for me when I tested it.

I just want to avoid commit churn as different devs environments fight over "the one true import style" (something that `go fmt` was _supposed_ to solve...)